_posts/2017-12-23-crystal-0.24.1-released.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Crystal 0.24.1 released!
+summary: Crystal has a new development release.
+thumbnail: +
+author: straight-shoota
+release_notes: true
+---
+
+[Crystal 0.24.1](https://github.com/crystal-lang/crystal/releases/tag/0.24.1) has been released!
+
+As any release it includes numerous bugfixes, cool features and performance improvements - in [387 commits since 0.23.1](https://github.com/crystal-lang/crystal/compare/0.23.1...0.24.1).
+
+0.24.0 was a preview release so this is the canonical realease following 0.23.1.
+
+# Exciting Changes
+
+* With `ThinLTO`, LLVM 4.0 and above allows faster release builds.
+* `SecureRandom` is renamed to `Random::Secure` and `SecureRandom.uuid` is replaced by a new type: `UUID`.
+* `BigDecimal` offers arbitrary precision, exact decimal numbers, supplementing `BigInt` and `BigFloat` which all are required by `require "big"`.
+* `Set` instances can be used as condition for `case`, matching when the case variable is included in the set.
+* `Time::Format` more robustly supports parsing sub-second precision.
+* `Tempfile` supports custom extensions.
+* The docs generator outputs documentation as JSON which can be easily parsed and used for external applications. The output directory of the docs generator can be configured using `--output` flag.
+* Garbage collection can be disabled using compiler flag `-Dgc_none`.
+* The default hash function uses an algorithm based on FunnyHash which is resiliant against DoS attacks.
+* `HTML.escape` only escapes HTML special characters `&<>"'`. `HTML.escape` recognizes all HTML5 named entities.
+* Multiple assignment can no longer be used to create an array (`a = 1, 2, 3`).
+* `skip` macro method was renamed to `skip_file`.
+* Packages are built with LLVM 3.9.1 which should hopefully fix issues with LLVM error when compiling to release mode.
```

Comment:
I'm pretty sure they're built with LLVM 4.

Reply:
Ah yes, that was from 0.24.0. I merged them together because I don't think we need a blog post for the pre-release (at least not after the real release).